menu.cpp

```diff
@@ -355,7 +357,9 @@ void siprintbinary(char* buffer, size_t const size, void const * const ptr)
 	unsigned char *b = (unsigned char*)ptr;
 	unsigned char byte;
 	int i, j;
-	memset(buffer, '\0', sizeof(buffer));
```

Comment:
The buffer is being memset with the size of a pointer instead of the size of the buffer.

Reply:
It's dead code left from original MiST version. 
The whole function must be removed.